packages/vertexai/lib/constants.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2016-present Invertase Limited & Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this library except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import { version } from './version';
+
+export const VERTEX_TYPE = 'vertexAI';
+
+export const DEFAULT_LOCATION = 'us-central1';
+
+export const DEFAULT_BASE_URL = 'https://firebasevertexai.googleapis.com';
+
+// This is the default API version for the VertexAI API. At some point, should be able to change when the feature becomes available.
+// `v1beta` & `stable` available: https://cloud.google.com/vertex-ai/docs/reference#versions
+export const DEFAULT_API_VERSION = 'v1beta';
+
+export const PACKAGE_VERSION = version;
+
+export const LANGUAGE_TAG = 'gl-js';
```

Comment:
Can you use another tag here? This is going to log all usage from this package as if it came from the JS SDK package. The full code for the headers is in `request.ts` if you want to customize it there too.

Reply:
I talked to the backend team and Cynthia and it sounds like `gl-rn` should work.

Reply:
Thanks, I've updated 👍